ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1032,6 +1032,7 @@ ethernet_interfaces:
     logging:
       event:
         link_status: < true | false >
+        congestion_drops: < true | false >  # not supported on vEOS-lab
```

Comment:
Please remove the comment regarding vEOS-LAB support, it isn't something we track in the documentation at this time.
```suggestion
        congestion_drops: < true | false >
```